sktime/forecasting/sarimax.py

```diff
@@ -221,3 +221,47 @@ def _extract_conf_int(prediction_results, alpha) -> pd.DataFrame:
         conf_int.columns = ["lower", "upper"]
 
         return conf_int
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        return [
+            {
+                "order": (4, 0, 0),
+            },
+            {
+                "order": (1, 1, 1),
+                "seasonal_order": (1, 1, 0, 3),
+            },
+            {
+                "order": (2, 1, 2),
+                "trend": "ct",
+                "enforce_stationarity": False,
+                "enforce_invertibility": False,
+                "concentrate_scale": True,
+                "use_exact_diffuse": True,
+            },
+            {
+                "order": [1, 0, 1],
```

Comment:
Why do you uses here lists instead of tuples? In the previous dicts you are using tuples.

Reply:
It is actually intentional :-)

The docs describe the orders as iterables, not tuple/list. So either should go and it is good to try.

Reply:
yes, better test coverage! both tuple and list case are tested this way!